bff/pkg/server/server.go

```diff
@@ -80,11 +78,11 @@ func (s server_imp) Run() error {
 	r.Use(CORS)
 
 	r.Get("/api/v1/activity", activity_handler(s.metrics_service))
-	r.Get("/api/v1/instance", info_handler(s.metrics_service, s.dbIdentifiers))
-	r.Get("/api/v1/instance/{dbIdentifier}/database", databases_handler(s.metrics_service))
+	r.Get("/api/v1/instance", info_handler(s.metrics_service))
+	r.Get("/api/v1/instance/database", databases_handler(s.metrics_service))
```

Comment:
should we be removing dbIdentifer as a route param here ? we just added it like this the other day

Reply:
The databases returned are specific to the dbinstance, so we need to revert line 82's changes. It should be `r.Get("/api/v1/instance/{dbIdentifier}/database", databases_handler(s.metrics_service))`

Reply:
As the `{dbIdentifier}` has `/`s in it, the routing doesn't work. That's why, I turned it into a normal parameter.